src/main/webapp/src/app/login-signup-page/login-signup.component.ts

```diff
@@ -81,13 +119,15 @@ export class LoginComponent {
    *
    * @param data object holding user data that will be used as a param in the post request
    */
-  postData(data: any): void {
+  postData(data: User): void {
     const httpOptions = {
       headers: new HttpHeaders({
```

Comment:
Is the `headers:` attribute necessary? I post JSON data to another servlet without entering this field (though we might deserialize the JSON data differently)

Reply:
No its not  I took it out.